### PR TITLE
[Misc] Fix mutation threshold in commons-xml

### PR DESCRIPTION
**Pull request by surli**
* Add a new test class for AbstractHTMLFilter

### CODE REVIEW

File: xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/html/filter/AbstractHTMLFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+import org.xwiki.xml.html.HTMLCleaner;
+import org.xwiki.xml.internal.html.DefaultHTMLCleaner;
+import org.xwiki.xml.internal.html.filter.AttributeFilter;
+import org.xwiki.xml.internal.html.filter.BodyFilter;
+import org.xwiki.xml.internal.html.filter.FontFilter;
+import org.xwiki.xml.internal.html.filter.LinkFilter;
+import org.xwiki.xml.internal.html.filter.ListFilter;
+import org.xwiki.xml.internal.html.filter.ListItemFilter;
+
+import static org.junit.Assert.assertTrue;
```
Comment (vmassol) [inline xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/html/filter/AbstractHTMLFilterTest.java]: This is not correct. Shoud be jupiter

**Comment by vmassol** [comment]
I would have written the test differently, just testing the filter and not htmlcleaner.

**Comment by vmassol** [comment]
Actually I think I would have written unit tests for the extending filters instead.

File: xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/html/filter/AbstractHTMLFilterTest.java (hunk 1 above)
Comment (surli) [reply xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/html/filter/AbstractHTMLFilterTest.java]: Fixed.

**Pull request merged**